lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -44,7 +46,7 @@
                 },
 
                 render: function () {
-                    this.$el.html(_.template(teamsTemplate));
+                    HtmlUtils.setHtml(this.$el, HtmlUtils.HTML(teamsTemplate));
```

Comment:
@andy-armstrong No, I think it's fine. So teamsTemplate is a string that is actually HTML and we don't want to interpolate anything into it, so there's no need to make a template and render it with context. `HtmlUtils.HTML` takes care of the escaping.